src/commands/buildCommands/trn.js

```diff
@@ -68,53 +68,53 @@ const writeTrnModules = messagesByLocale => ({ filename, msgids }) => {
 const componentTrnDefinitions$ = localTrns$.map(trnsFromFile => ({
 	filename: path.resolve(
 		paths.repoRoot,
-		trnsFromFile[0].file.replace(/\.jsx?$/, '')
+		trnsFromFile[0].file.replace(/\.jsx?$/, "")
 	),
-	msgids: trnsFromFile.map(({ id }) => id),
+	msgids: trnsFromFile.map(({ id }) => id)
 }));
 
 // Function for importing Flatpickr (fp) locale data for a particular 2-character
 // language code code - see https://flatpickr.js.org/localization/
 const fpLocale = lang => {
 	const langFile = require(require.resolve(`flatpickr/dist/l10n/${lang}`, {
-		paths: [paths.repoRoot],
+		paths: [paths.repoRoot]
 	}));
 	if (!langFile) {
 		console.error(
 			`Flatpickr locale data for language ${lang} cannot be returned`
 		);
 	}
-	return langFile;
+	return langFile.default[lang];
```

Comment:
Nice find!!!!

Reply:
Just for context, it looks like this bug appeared from [this major version update of `flatpickr` in mup-web](https://github.com/meetup/mup-web/commit/426903eab18ff74fc9e307ca7bacc015ed9ae68a#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L86) which included [changes to how locale data is exported](https://github.com/flatpickr/flatpickr/releases/tag/v4.0.0)

Unfortunately this kind of thing is pretty tricky to write unit tests for, but since mup-web defines the `flatpickr` dependency, it _probably_ would be slightly better to have this module-parsing logic in mup-web's [DatePicker.jsx](https://github.com/meetup/mup-web/blob/master/src/components/forms/DatePicker.jsx#L54) to avoid this change breaking other applications that might still be using flatpickr v3, e.g.

```
const localeOptions = pickerLocale[intl.locale].default[[intl.locale];
```